user_management/ui/tests/test_views.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.auth.tokens import default_token_generator
+from django.core.exceptions import PermissionDenied
+from django.http import Http404
+from django.test import override_settings
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+
+from user_management.models.tests.factories import VerifyEmailUserFactory
+from user_management.models.tests.models import VerifyEmailUser
+from user_management.models.tests.utils import RequestTestCase
+from .. import views
+
+
+@override_settings(AUTH_USER_MODEL='tests.VerifyEmailUser')
```

Comment:
Django 1.6 doesn't have `override_settings`. I suggest we take this opportunity to drop support for 1.6 and 1.7.